stock_valued_picking_report/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Pedro M. Baeza - Tecnativa <pedro.baeza@tecnativa.com>
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2016 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    valued_picking = fields.Boolean(
+        string='Valued picking',
```

Comment:
Odoo will give this exact string automatically to the field based on the field technical name if you don't define it (converts `valued_picking` into `Valued picking`), so not much sense on defining that manually IMHO.